src/main/java/com/appdynamics/extensions/webspheremq/metricscollector/MetricsCollectorContext.java

```diff
@@ -10,12 +10,17 @@
 import com.ibm.mq.headers.pcf.PCFException;
 import com.ibm.mq.headers.pcf.PCFMessage;
 import com.ibm.mq.headers.pcf.PCFMessageAgent;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import javax.annotation.Nullable;
 import javax.annotation.concurrent.Immutable;
 import java.io.IOException;
 import java.util.*;
 import java.util.concurrent.Future;
```

Comment:
import java.util.concurrent.Future  is not used.    but it's ok, i have created a change locally.